rtsp.go

```diff
@@ -53,6 +53,12 @@ type Config struct {
 	DistortionParams *transform.BrownConrady            `json:"distortion_parameters,omitempty"`
 }
 
+// CodecFormat contains a pointer to a format and the corresponding FFmpeg codec.
+type CodecFormat struct {
```

Comment:
lets make this type & its members private please as we don't expect them to be used outside of the viamrtsp package.